app/models/topic.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id          :bigint           not null, primary key
+#  description :string(50)       not null
+#  image       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class Topic < ApplicationRecord
+  validates :description, presence: true, uniqueness: true
```

Comment:
maybe, is a good idea to add a validation to check the length of the description